src/ws/metrics.rs

```diff
@@ -99,14 +99,14 @@ pub async fn validate_bearer_token(
 }
 
 lazy_static! {
-    pub static ref ERROR_RESPONSE_CODE_COLLECTOR: IntCounterVec = {
+    pub static ref PROCEDURE_CALL_COLLECTOR: IntCounterVec = {
         let opts = Opts::new(
-            "dcl_social_service_rpc_error_response_code",
-            "Social Service RPC Websocket Error Response Codes",
+            "dcl_social_service_rpc_procedure_call",
```

Comment:
```suggestion
            "ws_rpc_procedure_call",
```

Reply:
mm, okay, but aren't those for business logic metrics only?

Reply:
I don't know, there is no such classification in the docs. We use the same naming convention for the current REST API: 
+ `dcl_social_service_http_requests_duration_seconds_bucket`
+ `dcl_social_service_http_requests_duration_seconds_count`
+ `dcl_social_service_http_requests_duration_seconds_sum`
+ `dcl_social_service_http_requests_total`